index.js

```diff
@@ -393,21 +393,28 @@ module.exports = class Hypercore extends EventEmitter {
 
   _oncoreupdate (status, bitfield, value, from) {
     if (status !== 0) {
+      const truncated = (status & 0b10) !== 0
+      const appended = (status & 0b01) !== 0
```

Comment:
Slightly unrelated: Should we store those flags in constants?

Reply:
They are just internal and only used here so unsure if worth the complexity. The simpler form might be to use an explicit map instead tho